homeassistant/components/zwave_js/services.yaml

```diff
@@ -71,9 +71,15 @@ refresh_value:
   name: Refresh value(s) of a Z-Wave entity
   description: Force update value(s) for a Z-Wave entity
   target:
```

Comment:
Should the target item still be present?

Reply:
nope that was a miss